Server/JokesOnYou.Web.Api/DTOs/TagReplyDto.cs

```diff
@@ -7,5 +7,11 @@ namespace JokesOnYou.Web.Api.DTOs
 {
     public class TagReplyDto
     {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public DateTime Created { get; set; }
+        public List<JokeReplyDto> Jokes { get; set; }
+        public UserReplyDTO Owner { get; set; }
```

Comment:
Won't this result in a security issue where the API provides to much data to the client (like token) / email address? Might be better to create a separate DTO for this? TagOwnerDTO with only id and username.

Reply:
I actually dont know the practices of DTOs and their naming convention and what is supposed to return what. I saw this Tag DTO already created and assumed it was intended

Reply:
dto = data transfer object. its also known as ViewModel because its just to give the correct data back to our client